style.css

```diff
@@ -325,5 +324,147 @@
     padding-top: 1rem;
     font-weight: bold;
   }
-  
+
+  #form {
+    display: flex;
+    flex-direction: column;
+    background: #6070ff;
+    width: 400px;
+    padding: 1em;
+    border: 1px solid #fff;
+    border-top-left-radius: 5rem;
+    background-image: url(images/contact\ form\ shapes-mobile2.svg);
+    background-repeat: no-repeat;
+    background-position: right -1px top 10px;
+    padding-bottom: 1px;
+  }
+
+  #msg {
+    min-width: 300px;
+    min-height: 170px;
+  }
+
+  #msg::placeholder {
+    color: #b3bac5;
+  }
+
+  .form-Ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .form-Li {
+    margin-top: 0.2rem;
+    border-radius: 3rem;
+  }
+
+  label {
+    /* Uniform size & alignment */
+    display: inline-block;
+    width: 90px;
+    text-align: right;
+  }
+
+  input,
+  textarea {
+    font-family: "Poppins", Arial, sans-serif;
+    width: 300px;
+    box-sizing: border-box;
+    border: 1px solid #999;
+    border-radius: 0.5rem;
+    padding: 10px;
+  }
```

Comment:
- [ ] You have still to fix the width of your form element because it is not like in the original design. I can suggest you use `width: 100%` for input and texture, and everything will look very pretty 👍
> Your version
<img width="484" alt="Screen Shot 2022-02-17 at 01 23 37" src="https://user-images.githubusercontent.com/49064106/154359493-e6cc6f86-a700-47dc-b5dc-90bc03a04c47.png">
 
> After I fixed the width to `100%`
<img width="762" alt="Screen Shot 2022-02-17 at 01 25 04" src="https://user-images.githubusercontent.com/49064106/154359809-3a85ca28-26e0-484b-a9cb-bfc95c1bb426.png">

Reply:
added